README.md

```diff
@@ -19,7 +19,7 @@ scale-up, operation and troubleshooting of innovative, advanced energy systems.
 
 ## Getting Started
 
-Our [complete documentation is online](https://idaes-pse.readthedocs.io/en/stable/) but here is a summarized set of steps to get started using the framework.
+Our [complete documentation is online](https://idaes-pse.readthedocs.io/en/stable/) but here is a summarized set of steps to get started using the framework. Please also visit the [IDAES PSE Discussions Board](https://github.com/IDAES/idaes-pse/discussions).

```

Comment:
This phrasing is a bit strange.  _Why_ are we asking to visit the discussions?  To get help / assistance? To see examples of how other users got through problems?